src/cf-core/usage.md

```diff
@@ -245,32 +242,30 @@ Adds a `.lt-ie8` fallback to hack inline block for IE 7 and below.
 
 Force word breaks within an element. Useful for small containers where text may over-run the width of the container.
```

Comment:
I'm going to hold off on this to hopefully avoid conflicts with the standardized cf-core `usage.md` file that has been merged into `canary`